src/components/EditMove.js

```diff
@@ -3,26 +3,42 @@ import React from "react";
 const EditMove = () => {
   return (
     <div className="editmove">
-      <h3>Edit Move</h3>
+      <div className="editmove_header_wrapper">
+        <h1>Edit Move</h1>
+      </div>
       <form className="editmove-wrapper">
         <label>
```

Comment:
We do not have label texts any more so we can also remove label-tags

Reply:
I agree with Tero that the label taga could be removed.